src/platform/wearables/wearable.ts

```diff
@@ -4,33 +4,165 @@ import {
+}
 
 /** @alpha */
-export type Wearable = BaseWearable & {
-  rarity: Rarity
-  collectionAddress: string
+export type StandardWearable = Omit<
+  WithRequired<Wearable, 'collectionAddress' | 'rarity'>,
+  'merkleProof'
+>
+
+export type ThirdPartyWearable = Omit<
+  WithRequired<Wearable, 'merkleProof'>,
+  'rarity' | 'collectionAddress'
+>
+
+const validateThirdParty = (wearable: Wearable) => {
+  if (!MerkleProof.validate(wearable.merkleProof)) return false
+  if (wearable.merkleProof.hashingKeys.length === 0) return false
+  const containsAllKeys = wearable.merkleProof.hashingKeys.every((key) =>
+    wearable.hasOwnProperty(key)
+  )
+
+  const proofIsNotEmpty = wearable.merkleProof.proof.length > 0
+
+  return containsAllKeys && proofIsNotEmpty
 }
 
+const validateStandardWearable = (
+  rarity?: Rarity,
+  collectionAddress?: string
+) => Rarity.validate(rarity) && !!collectionAddress
+
+export const isStandard = (wearable: Wearable): wearable is StandardWearable =>
+  validateStandardWearable(wearable.rarity, wearable.collectionAddress)
+
+export const isThirdParty = (
+  wearable: Wearable
+): wearable is ThirdPartyWearable => validateThirdParty(wearable)
+
 /** @alpha */
 export namespace Wearable {
   export const schema: JSONSchema<Wearable> = {
-    ...BaseWearable.schema,
+    type: 'object',
     properties: {
-      ...BaseWearable.schema.properties!,
+      ...displayableProperties,
+      id: {
+        type: 'string'
+      },
+      descriptions: {
+        type: 'array',
+        items: I18N.schema,
+        minItems: 1
+      },
       collectionAddress: {
+        type: 'string',
+        nullable: true
+      },
+      rarity: {
+        ...Rarity.schema,
+        nullable: true
+      },
+      names: {
+        type: 'array',
+        items: I18N.schema,
+        minItems: 1
+      },
+      data: {
+        type: 'object',
+        properties: {
+          replaces: {
+            type: 'array',
+            items: WearableCategory.schema
+          },
+          hides: {
+            type: 'array',
+            items: WearableCategory.schema
+          },
+          tags: {
+            type: 'array',
+            items: {
+              type: 'string',
+              minLength: 1
+            }
+          },
+          representations: {
+            type: 'array',
+            items: WearableRepresentation.schema,
+            minItems: 1
+          },
+          category: WearableCategory.schema
+        },
+        additionalProperties: false,
+        required: ['replaces', 'hides', 'tags', 'representations', 'category']
+      },
+      thumbnail: {
+        type: 'string'
+      },
+      image: {
         type: 'string'
       },
-      rarity: Rarity.schema
+      metrics: {
+        ...Metrics.schema,
+        nullable: true
+      },
+      merkleProof: {
+        ...MerkleProof.schema,
+        nullable: true
+      },
+      content: {
+        type: 'object',
+        nullable: true
+      }
```

Comment:
Content can be typed as a string -> string map:
```suggestion
      content: {
        type: 'object',
        additionalProperties: { type: 'string' },
        required: []
      },
```